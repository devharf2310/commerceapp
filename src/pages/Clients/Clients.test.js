import React from 'react'
import renderer from 'react-test-renderer'
import Clients from './Clients';

describe('clients snapshot tests', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Clients/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
