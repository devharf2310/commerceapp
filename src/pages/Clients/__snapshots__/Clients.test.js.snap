// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clients snapshot tests renders correctly 1`] = `
<h1>
  Clients
</h1>
`;
